test/spec/provider/camunda/ConnectorSpec.js

```diff
@@ -1083,7 +1082,7 @@ function getOutputParameterGroupLabel(container) {
 }
 
 function getParameterNameInput(container) {
-  return domQuery('input[id*="parameterName"]', getInputParametersGroup(container));
+  return domQuery('input[name="name"]', getInputParametersGroup(container));
```

Comment:
Why is this change needed?

Reply:
There is no factual reason. I worked on this and played with this and accidentally rewrote it. But the result is the same. I will revert the change so to not cause confusion.